packages/cdk8s-cli/README.md

```diff
@@ -18,6 +18,8 @@ To install the CLI use one of the following methods:
 * npm: `npm i -g cdk8s-cli`
 * yarn: `yarn global add cdk8s-cli`
 
+| Note: the latest version available via Homebrew is `0.33.0` because Homebrew does not support beta releases. Please use `npm` or `yarn` if you need a later version of the `cdk8s` cli.
```

Comment:
```suggestion
> Note: the latest version of the `cdk8s` cli available via Homebrew is `0.33.0` because Homebrew does not accept beta versions. Once a stable `1.0.0` version is released it will be available on Homebrew as well. In the meantime, please use `npm` or `yarn` to install the latest cli.
```